test/interpreter/function-stdevp.spec.ts

```diff
@@ -0,0 +1,22 @@
+import {ErrorType, HyperFormula} from '../../src'
+import {adr, detailedError} from '../testUtils'
+import {ErrorMessage} from '../../src/error-message'
+
+describe('Function STDEV.P', () => {
+  it('should take at least one argument', () => {
+    const engine = HyperFormula.buildFromArray([
+      ['=STDEV.P()'],
+    ])
+
+    expect(engine.getCellValue(adr('A1'))).toEqual(detailedError(ErrorType.NA, ErrorMessage.WrongArgNumber))
+  })
+
+  it('should calculate standard deviation (population)', () => {
+    const engine = HyperFormula.buildFromArray([
+      ['=STDEV.P(2, 3)'],
+      ['=STDEV.P(B2:D2, E2, F2)', 2, 3, 'foo', 'bar', 4],
+    ])
+    expect(engine.getCellValue(adr('A1'))).toEqual(0.5)
+    expect(engine.getCellValue(adr('A2'))).toBeCloseTo(0.816496580927726, 6)
+  })
+})
```

Comment:
Tests are missing as in other cases but here I have to ask which version of STDEV.P we implement?

ODFF has STDEVP only, but Excel has both STDEVP and STDEV.P. One is improved, other for backward compatibility. How does that relate to the ODFF one? ODFF has the old one or already the improved version?

Reply:
The STDEV.P is just a newer version of STDEVP and STDEV functions of excel. There is no difference between them.

Reply:
Cool, then nothing to do. Pitty we don't have function name aliases. We could map STDEVP to STDEV.P and be compatible with ODFF and Excel